### ISSUE TITLE
Security Issue due to lsass handle

### ISSUE THREAD

**Issue by k4nfr3** [zeronetworks/rpcfirewall#29]
Hello,

I stumbled on this issue by accident.
rpcFwManager.exe has a full control handle on LSASS, which can be **abused to dump LSASS** via a DeDup handle attack.

Maybe one should change this level of granted access to the bare minimum needed instead ?

![image](https://user-images.githubusercontent.com/45294916/203993279-21121909-d5ee-4a05-948c-5d747ad8c3e4.png)

**Comment by k4nfr3** [comment]
I guess it's the pre-requesite to be able to hook the RPC calls, that's why it has an handle for everysingle process running on the machine. But this also means that if the machine has RunAsPPL activated, as RPCFirewall software is not signed by Microsoft, it cannot block RPC calls to LSASS anymore as it can't have a handle on lsass.exe process aka \PIPE\lsarpc for example with the PetitPotam MS_EFSR

**Comment by sagiesec** [comment]
Yes. The access to RPC server processes (including lsass.exe) is by design. I'm not sure it's a security issue, as any other administrative process can do the same if you can run a privileged process on a host. 
Regarding PPL - you are also right. The walkaround for such cases is to use the RPC filters mechanism (which is also supported by RpcFwManager) to block specific UUIDs exposed by lsass.
